### PR TITLE
Added two methods to PlayerManager

### PR DESCRIPTION
**Pull request by shieldbug1**
1.8 version of #1490

### CODE REVIEW

File: patches/minecraft/net/minecraft/server/management/PlayerManager.java.patch [hunk 1]
```diff
@@ -1,6 +1,26 @@
 --- ../src-base/minecraft/net/minecraft/server/management/PlayerManager.java
 +++ ../src-work/minecraft/net/minecraft/server/management/PlayerManager.java
-@@ -122,15 +122,24 @@
+@@ -1,9 +1,11 @@
+ package net.minecraft.server.management;
+ 
+ import com.google.common.collect.Lists;
++
```
Comment (Parker8283) [inline patches/minecraft/net/minecraft/server/management/PlayerManager.java.patch]: Don't add lines

Comment (shieldbug1) [inline patches/minecraft/net/minecraft/server/management/PlayerManager.java.patch]: Must've been my IDE, whoops. I'll fix this up in a moment.

**Comment by shieldbug1** [comment]
Fixed patch.

**Comment by diesieben07** [comment]
Any news on this?

**Comment by cpw** [comment]
I think I'm gonna do this slightly differently.

**Comment by williewillus** [comment]
Is this still needed? afaik at least in 1.9 you can do this directly yourself

**Comment by diesieben07** [comment]
No, it's not needed anymore. The inner class is now public.

**Comment by Actuarius** [comment]
@williewillus closed this issue

**Pull request closed without merging**